spikeinterface/comparison/multicomparisons.py

```diff
@@ -218,7 +218,7 @@ def __init__(self, sampling_frequency, multisortingcomparison,
             for k in ('agreement_number', 'avg_agreement', 'unit_ids'):
                 values = [self._msc._new_units[unit_id][k]
                           for unit_id in unit_ids]
-                self.set_property(k, values, ids=unit_ids)
+                self.set_property(k, values)
```

Comment:
Can you put back this line, it should work now. And maybe this unit_ids was here because a diffrents order.